src/scripts/validation.js

```diff
@@ -66,7 +66,7 @@ const resetValidation = (formElement, inputList, settings) => {
 
     const setEventListeners = (formElement, config) => {
     const inputList = Array.from(formElement.querySelectorAll(config.inputSelector));
-    const buttonElement = formElement.querySelector(config.submitButtonSelector);
+    const buttonElement = formElement.querySelector(config.submitButtonSelector) || formElement.querySelector(".modal__delete-button");
```

Comment:
` || formElement.querySelector(".modal__delete-button");` is not needed inside `validation.js`. You don't need to validate the **delete** form because it doesn't have any inputs